keras_nlp/src/models/gemma/gemma_backbone_test.py

```diff
@@ -132,7 +132,40 @@ def test_distribution(self):
                 self.assertEqual(
                     tuple(w.value.sharding.spec), ("batch", "model")
                 )
-            if "ffw_linearl" in w.path:
+            if "ffw_linear" in w.path:
                 self.assertEqual(
                     tuple(w.value.sharding.spec), ("model", "batch")
                 )
+
+    def test_distribution_with_lora(self):
+        if keras.backend.backend() != "jax":
+            return
+        devices = keras.distribution.list_devices("CPU")
+        if len(devices) == 1:
+            # Need more than 1 device for distribution testing.
+            return
```

Comment:
Same here, `self.skipTest`. Also not that this will never run on any CI we have today.

Reply:
This one is not mine. I did not touch it. Indeed, I noticed this would not run on any CI today. I suppose we do not have a "fake distributed system" with 8 CPUs on which to run this?
To verify the tests, I had to manually change "CPU" to "TPU" on my test machine. How can we fix this?

Reply:
Looks like we might be able to do `xla_force_host_platform_device_count` before we launch the test runner.

This would at least give us some coverage for single device distribution, but not multi-device.

I'll pull this in just to keep moving, but will look at doing that as a follow up. cc @SamanehSaadat since you are looking at model parallel.